GeoNewsFinder4/components/RelatedArticles.js

```diff
@@ -1,17 +1,67 @@
-import React from 'react';
-import { View, StyleSheet } from 'react-native'; 
+import { React, useEffect, useState } from 'react';
+import { View, Text, TouchableOpacity, StyleSheet, FlatList, Image } from 'react-native';
+import { Card } from 'react-native-elements';
+import getAPIdata from '../utils/getAPIdata';
 
-const RelatedArticlesRoute = () => {
+const RelatedArticlesRoute = ({ route }) => {
+
+  const [newsData, setData] = useState([]);
+  useEffect(() => {
+    const fetchData = async () => {
```

Comment:
might be more efficient to declare this fetchData outside of the use effect so we don't have to "generate" a function variable fetchData every time useEffect gets called.